src/Dist_Archive_Command.php

```diff
@@ -416,39 +414,53 @@ protected function is_path_contains_symlink( $path ) {
 	}
 
 	/**
-	 * Check a file from the plugin against the list of rules in the `.distignore` file.
+	 * Filter all files in a path to list of file to include or to exclude from the archive.
```

Comment:
This sentence seems to be grammatically wrong and needs to be rephrased.

Reply:
> Filter all files in a path to either: a list of files to include, or a list of files to exclude, from the archive.